fixtures/g-cloud-6-iaas-service.json

```diff
@@ -0,0 +1,356 @@
+{
```

Comment:
As mentioned above, these lots are redundant so we may need new fixtures, or another way of creating g-cloud services.